darkflow/net/flow.py

```diff
@@ -115,15 +115,10 @@ def predict(self):
         to_idx = min(from_idx + batch, len(all_inps))
 
         # collect images input in the batch
-        inp_feed = list(); new_all = list()
         this_batch = all_inps[from_idx:to_idx]
-        for inp in this_batch:
-            new_all += [inp]
-            this_inp = os.path.join(inp_path, inp)
-            this_inp = self.framework.preprocess(this_inp)
-            expanded = np.expand_dims(this_inp, 0)
-            inp_feed.append(expanded)
-        this_batch = new_all
+        inp_feed = pool.map(lambda inp: (
```

Comment:
Where is this package pool comes from? do we have to import it? install?

Reply:
pool variable was initialized at the beginning of flow.py which was already used in ThreadPool batch output few months ago.

Reply:
pool variable is initialized to Python built-in ThreadPool at the beginning of flow.py which has been used in ThreadPool batch output for prediction few months ago. 😎

Reply:
"pool" variable is initialized as a Python built-in ThreadPool instance at flow.py#L15 which has been used in ThreadPool batch output for prediction few months. 😎

Reply:
"pool" variable is initialized as a Python built-in ThreadPool instance at flow.py#L15 which has been used few months in ThreadPool batch post-processing for prediction. 😎